tests/metachecks/explicit-regex-capture.rule.yaml

```diff
@@ -0,0 +1,11 @@
+rules:
```

Comment:
I think you've inverted the use of the .rule.yaml vs .yaml.
Look at the other test file in the directory (I have to admit it's a but confusing, so some renaming could be useful or
a README).
Also the other tests have some     # ruleid: semgrep-metacheck-builtin
in the .rule.yaml to actually test the functionality but none of your test have one.

Reply:
I'm trying to test that the built-in metachecks don't fire. All of these test rules should be valid.

Reply:
Improved the comment for clarity